code/src/App.js

```diff
@@ -1,9 +1,83 @@
-import React from 'react'
+import React, {useState, useEffect} from 'react'
+
+import { Fetch_API, like_url } from './reusable/urls'
+
+import { CardContainer } from './components/CardContainer'
+import { Form } from './components/Form'
 
 export const App = () => {
+  const [thoughtsList, setThoughtsList] = useState([])
+  const [thoughtsNew, setThoughtsNew] = useState('')
+  const [loader, setLoader] = useState (true)  
```

Comment:
Your loader seems to work it's just quick, not sure if there is  a way to inspect / pause it with devtools animation feature, was trying this from here but couldn't figure it out: https://css-tricks.com/inspecting-animations-in-devtools/